ggshield/utils.py

```diff
@@ -147,6 +147,8 @@ def get_lines_from_patch(content: str, filemode: Filemode) -> Iterable[Line]:
             line_pre_index = pre_index
             line_content = line[1:]
             category = LineCategory.deletion
+        elif line_type == "\\":
+            line_content = line[1:]
```

Comment:
This makes sense, but can you add a test for it?

Reply:
Added a supplementary test case for the `make_indices`.
There is no test yet for the `get_lines_from_file` function.

Reply:
Looks good to me :+1:.

Reply:
Actually something is odd: codecov says [your code is not reached by the test suite](https://app.codecov.io/gh/GitGuardian/ggshield/compare/137/diff?src=pr&el=tree). Can you look into this?

Reply:
Seems fixed with latest changes